heroku/cli
# chore(cli): add nov 28 updates

This PR has all the relevant updates required to allow the cli to successfully run post nov 28. 

How to test:

1. Enable `restrict-free-dynos`.
2. Pull down this branch and within the root directory, locally run the commands by using `./bin/run <command>`.
3. Check out the `--help` documentation for:
     - addons
     -  dynos
     - apps
     Make sure documentation examples reference the new `mini`, `eco` and `basic` options within their respective commands
5. Try passing `free` in `--size` flags for the `ps` commands.
6. Verify the inability to scale to free dynos or successfully pass `free` in the `--size` flags
7. Look around for any other potential bugs or breaks!
